### ISSUE TITLE
Multiple xp and damage delay

### ISSUE THREAD

**Issue by rockfireredmoon** [rockfireredmoon/iceee#164]
Reported By: Dawne Evershield
Category: MOB

The frostbound maris hero's have a bug that  when you kill him, they give an exp reward of the standard exp amount but times 1-6  times. Also according to liska, they have a damage delay of an amount times 1-6 times.

**Comment by LiskaQuicksilver** [comment]
I can indeed verify this bug. The normal Frostbound Marid Heros (still testing the ones in Grim's Lair) tend to do this AFTER they have been killed and usually a few seconds following the kill. Oftentimes the player gets the allotted XP for their XP level, but, then it adds at random between 0-5 additional XP rewards. For example with Liska at level 50, I get 80XP per kill initially, then afterward, and at random, an additional 80XP 0-5 times.

It also tends to give delayed damage points between 1-10 times usually to the amount of 13 damage points each. Oddly the first two Hero kills I made rewarded me with 130XP each. This did not happen any time after.

Still testing the heroic varients seen in Grim's Lair to see if it, too, is affected or not.

For whatever reason, this bug affects the Frostbound Marid Heros ONLY. None of the other mobs in the area seem affected by it.

**Comment by rockfireredmoon** [comment]
Fixed, well .. I am not entirely sure if this is going to fix it or not, but I can no longer reproduce the damage problem.

The script used for this creature (rightly or wrongly custom_heathendel_rogue40), has been converted to Squirrel, and I took some time to tune the script speed system (quite different to TSL, but still relevant, I am trying to keep the numbers meaning roughly the same). 

A new global config parameter was added to set the overall speed queue speed. Scripts that wish to respect should use ai.exec() instead of ai.queue().

Awaiting feedback (upon Fridays update)
